bower.json

```diff
@@ -44,8 +44,7 @@
     "moment-timezone": "0.5.3",
     "ng-sortable": "1.3.4",
     "registry-image-widgets": "0.0.2",
-    "ui-select": "angular-ui-select#0.19.4",
-    "uri.js": "1.18.0"
+    "ui-select": "angular-ui-select#0.19.4"
   },
```

Comment:
There is still 'uri.js' in 'dependencies':

"dependencies": {
   ...
    "uri.js": "1.18.0"